README.md

```diff
@@ -26,6 +26,18 @@ suggested by the README.
 The addon files will be placed in `../../xbmc/kodi-build/addons` so if you build Kodi from source and run it directly 
 the addon will be available as a system addon.
 
+### Linux for regular users
+
+The following instructions assume you will have installed Kodi already in either /usr or /usr/local
```

Comment:
I doubt it is Kodi that is needed to be installed for this, actually it's one or more of it's dev packages (I don't know the details)? You should write here which packages are actually needed. I think kodi package alone will not be sufficient.

Reply:
The list of build-deps was never part of the README.
I fail to see how adding such a list suddenly has become my task.

Reply:
If we add instructions, they have to be clear, otherwise we will not improve anything, but add another source of confusion.

> I fail to see how adding such a list suddenly has become my task.

 Well, do what you like, but if you want this merged we all need to be friendly and constructive.